pkg/api/cluster_types.go

```diff
@@ -109,7 +109,8 @@ const (
 	ClusterProviderAwsEKS     ClusterProviderType = "aws_eks"
 	ClusterProviderStandalone ClusterProviderType = "standalone"
 
-	Enterprise ClusterType = "enterprise"
+	EnterpriseClusterType       ClusterType = "enterprise"
+	ManagedDataPlaneClusterType ClusterType = "managed"
```

Comment:
wdyt of naming the ClusterType to `DataPlaneClusterType` instead to make it more specific?

```suggestion
	EnterpriseDataPlaneClusterType       DataPlaneClusterType = "enterprise"
	ManagedDataPlaneClusterType DataPlaneClusterType = "managed"
```

Reply:
changed. I think it makes sense. There are couple other types that could use the same prefix but I'll leave those to some other PRs